packages/cli/src/telemetry.ts

```diff
@@ -0,0 +1,146 @@
+/// <reference lib="dom" />
+import fs from 'fs-extra';
+import uuid from 'uuid/v4';
+import path from 'path';
+import crypto from 'crypto';
+import envPaths from 'env-paths';
+import mapValues from 'lodash.mapvalues';
+import pickBy from 'lodash.pickby';
+import inquirer from 'inquirer';
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import 'firebase/firestore';
+import { DISABLE_INTERACTIVITY } from './prompts/prompt';
+
+import ProjectFile from './models/files/ProjectFile';
+
+const FIREBASE_CONFIG = {
+  apiKey: 'AIzaSyAZoGB2p26UejODezZPmczgwehI6xlSKPs',
+  authDomain: 'cli-telemetry.firebaseapp.com',
+  databaseURL: 'https://cli-telemetry.firebaseio.com',
+  projectId: 'cli-telemetry',
+  storageBucket: '',
+  messagingSenderId: '449985678611',
+  appId: '1:449985678611:web:88b411adc68e6521e19ee6',
+};
+
+type Field = string | number | boolean;
+
+interface GlobalTelemetryOptions {
+  optIn: boolean;
+  uuid: string;
+  salt: string;
+}
+
+export default {
+  async report(commandName: string, options: any, interactive: boolean): Promise<void> {
+    const telemetry = await checkOptIn(interactive);
+    if (telemetry === undefined || !telemetry.optIn) return;
+
+    // extract network name if present
+    let { network } = pickBy(options, (_, key) => key === 'network');
```

Comment:
Am I missing something or `let network = options.network;` works just fine?

Reply:
I believe you are right :man_facepalming:, ~We~ I was probably using `pickBy` to take more than one attribute, although it's not necessary in that case neither.